.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,8 +19,8 @@
 
 ### Before submitting the PR, please make sure you do the following
 
-- [ ] Read the [Contributing Guidelines](https://github.com/vitejs/vite/blob/main/CONTRIBUTING.md).
-- [ ] Read the [Pull Request Guidelines](https://github.com/vitejs/vite/blob/main/CONTRIBUTING.md#pull-request-guidelines) and follow the [PR Title Convention](https://github.com/vitejs/vite/blob/main/.github/commit-convention.md).
```

Comment:
I think we can remove the PR Title Convention part since it's written in the Pull Request Guideline and we also have a CI with error messages.

I merged this in a single checkbox to keep the number of checkboxes.